gazebo_plugins/src/gazebo_ros_imu.cpp

```diff
@@ -99,15 +99,6 @@ void GazeboRosIMU::LoadThread()
   else
     this->link_name_ = this->sdf->Get<std::string>("bodyName");
 
-  if (!this->sdf->HasElement("frameId"))
```

Comment:
We did add this, but they added the same parameter upstream with a different name. The default value is different, though.